### DISCUSSION TITLE
Different `output-dir` for different formats

### DISCUSSION THREAD

**Discussion by danielvartan** [quarto-dev/quarto-cli#7004] [Q&A] [answered]
### Description

Is there a way to set `output-dir`  by format?

Example. I may want to render the pdf format, keeping the tex file, in a folder named `pdf`. At the same time, I may render the `html` format in the repository `docs` folder.

**Comment by mcanouil** [comment, marked as answer]
Yes, via project profiles, see <https://quarto.org/docs/projects/profiles.html>.

**Comment by danielvartan** [comment]
Exactly what I needed. Thank you!
